src/zendoo/mcTestCall.cpp

```diff
@@ -360,11 +415,20 @@ void create_verify_test_csw_proof(std::string ps_type_raw, std::string csw_type_
     assert(nullifier_f != NULL);
     assert(ret_code == CctpErrorCode::OK);
 
-    // Parse mc_pk_hash
-    assert(IsHex(argv[arg]));
+    // Parse mc_address
+    // Extract pubKeyHash from the address
+    std::vector<unsigned char> vchData;
+    assert(DecodeBase58(argv[arg++], vchData));
     uint160 mc_pk_hash_vec;
-    mc_pk_hash_vec.SetHex(argv[arg++]);
-    assert(mc_pk_hash_vec.size() == 20);
+    memcpy(&mc_pk_hash_vec, &vchData[2], 20);
```

Comment:
Consider to change name mc_pk_hash_vec  to pk_dest as it implemented in create_verify_test_cert_proof function. The same for mc_pk_hash variable below.

Reply:
done